extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/GeographicRestrictionException.java

```diff
@@ -0,0 +1,11 @@
+package org.schabi.newpipe.extractor.exceptions;
+
+public class GeographicRestrictionException extends ParsingException {
```

Comment:
I'd have this extend `ContentNotAvailableException`, in order not to introduce breaking changes in the library